db/pf-schema-X.Y.Z.sql

```diff
@@ -1261,3 +1255,21 @@ CREATE TABLE auth_log (
   KEY pid (pid),
   KEY  attempted_at (attempted_at)
 ) ENGINE=InnoDB;
+
+--
+-- Creating chi_cache table
+--
+
+CREATE TABLE `chi_cache` (
+  `key` VARCHAR(767),
+  `value` LONGTEXT,
+  `expires_at` REAL,
```

Comment:
Downside is that the now() function doesn't exist in all storages which the driver supports

Using the UNIX timestamp was the best way I found to support the different storages and keep the code simpler

I'm open to another implementation but this one works well